src/python/dxpy/utils/executable_unbuilder.py

```diff
@@ -140,46 +151,60 @@ def make_cluster_bootstrap_script_file(region, entry_point, code, suffix):
-                if asset_record:
+    for region in info["runSpec"]["bundledDependsByRegion"]:
+        for dep in reversed(info["runSpec"]["bundledDependsByRegion"][region]):
+            file_handle = get_handler(dep["id"])
+            if isinstance(file_handle, dxpy.DXFile):
+                asset_record_id = file_handle.get_properties().get("AssetBundle")
+                if asset_record_id:
                     try:
-                        asset_json = {"name": asset_record.describe().get("name"),
-                                              "project": asset_record.get_proj_id(),
-                                              "folder": asset_record.describe().get("folder"),
-                                              "version": asset_record.describe(fields={"properties": True}
-                                                                               )["properties"]["version"]
-                                              }
+                        # asset_desc = dxpy.api.record_describe(asset_record_id, {"properties": True})
+                        # asset_json = {"name": asset_desc.get("name"),
+                        #             "project": asset_desc.get("project"),
+                        #             "folder": asset_desc.get("folder"),
+                        #             "version": asset_desc.get("properties", {}).get("version")
+                        #             }
+                        asset_json={"id": asset_record_id}
                         if dep.get("stages"):
                             asset_json["stages"] = dep["stages"]
-                        asset_depends.append(asset_json)
-                        deps_to_remove.append(dep)
+                        asset_depends[region].append(asset_json)
+                        deps_to_remove.add(dep["name"])
                     except DXError:
-                        print("Describe failed on the assetDepends record object with ID - " +
-                              asset_record_id + "\n", file=sys.stderr)
+                        print("Describe failed on the assetDepends record object with ID {} in region {}.\n".format(asset_record_id, region),
+                        file=sys.stderr)
                         pass
-            else:
-                break
-    # Reversing the order of the asset_depends[] so that original order is maintained
-    asset_depends.reverse()
+                else:
+                    break
+        # Reversing the order of the asset_depends[] so that original order is maintained
+        asset_depends[region].reverse()
     # resources/ directory
     created_resources_directory = False
     if not omit_resources:
-        for dep in info["runSpec"]["bundledDepends"]:
-            if dep in deps_to_remove:
+        if not enabled_regions:
+            raise DXError("Cannot download resources of the requested executable {} since it is not available in any of the billable regions.".format(info["name"]))
```

Comment:
How about adding something like: "You can run use the omit-resources flag to skip downloading the resources"